### ISSUE TITLE
tfjs-layers: Add Masking layer

### ISSUE THREAD

**Issue by caisq** [tensorflow/tfjs#1502]
#### TensorFlow.js version
1.0.4

**Comment by caisq** [comment]
See Python implementation at https://github.com/keras-team/keras/blob/master/keras/layers/core.py#L28

**Comment by Gurpreetsingh9465** [comment]
@caisq can I work on this issue?

**Comment by caisq** [comment]
@Gurpreetsingh9465 Yes, please! :) Thanks for offering to help.

**Comment by Gurpreetsingh9465** [comment]
@caisq implemented logic of masking layer 
`import * as tf from '@tensorflow/tfjs';`
`let a = tf.tensor([[[1,2,3],[1,2,3],[1,2,3]],[[8,8,8],[1,5,7],[8,8,9]],[[8,8,8],[8,8,8],[8,8,8]]]);`
`let mask_value = 8;`
`let test = tf.notEqual(a,mask_value);`
`let mask = tf.any(test,-1,true)`
`mask = mask.asType(a.dtype);`
`let res = a.mul(mask);`
`a.print();`
`res.print();`
should i create another file for masking layer and it's test cases inside this folder [src/layers](https://github.com/tensorflow/tfjs-layers/tree/master/src/layers)

**Comment by Gurpreetsingh9465** [comment]
@caisq did a PR regarding this issue tensorflow/tfjs-layers#525
